db/migrate/20170407082423_add_team_id_to_steps.rb

```diff
@@ -0,0 +1,114 @@
+require File.expand_path('app/helpers/database_helper')
+include DatabaseHelper
+
+class AddTeamIdToSteps < ActiveRecord::Migration
+  def up
+    if db_adapter_is? 'PostgreSQL'
+      # Removing old indexes
+      remove_index :projects, :name if index_exists?(:projects, :name)
+      remove_index :my_modules, :name if index_exists?(:my_modules, :name)
+      remove_index :my_modules, :description if index_exists?(:my_modules, :description)
+      remove_index :protocols, :name if index_exists?(:protocols, :name)
+      remove_index :protocols, :description if index_exists?(:protocols, :description)
+      remove_index :protocols, :authors if index_exists?(:protocols, :authors)
+      remove_index :protocol_keywords, :name if index_exists?(:protocol_keywords, :name)
+      remove_index :my_module_groups, :name if index_exists?(:my_module_groups, :name)
+      remove_index :tags, :name if index_exists?(:tags, :name)
+      remove_index :results, :name if index_exists?(:results, :name)
+      remove_index :result_texts, :text if index_exists?(:result_texts, :text)
+      remove_index :reports, :name if index_exists?(:reports, :name)
+      remove_index :reports, :description if index_exists?(:reports, :description)
+      remove_index :assets, :file_file_name if index_exists?(:assets, :file_file_name)
+      remove_index :samples, :name if index_exists?(:samples, :name)
+      remove_index :sample_types, :name if index_exists?(:sample_types, :name)
+      remove_index :sample_groups, :name if index_exists?(:sample_groups, :name)
+      remove_index :sample_custom_fields, :value if index_exists?(:sample_custom_fields, :value)
+      remove_index :steps, :name if index_exists?(:steps, :name)
+      remove_index :steps, :description if index_exists?(:steps, :description)
+      remove_index :checklists, :name if index_exists?(:checklists, :name)
+      remove_index :checklist_items, :text if index_exists?(:checklist_items, :text)
+      remove_index :tables, :name if index_exists?(:tables, :name)
+      remove_index :users, :full_name if index_exists?(:users, :full_name)
+      remove_index :comments, :message if index_exists?(:comments, :message)
+      remove_index :comments, :type if index_exists?(:comments, :type)
+      remove_index :protocols, :protocol_type if index_exists?(:protocols, :protocol_type)
+      remove_index :checklists, :step_id if index_exists?(:checklists, :step_id)
+
+      execute(
```

Comment:
I don't think it's worth it, as if we need to change regex in it, it will be needed to run migration again. Keeping this function in migration we will know which exact version of this function is added by this migration.